package.json

```diff
@@ -16,28 +16,25 @@
     "build": "lerna run build",
     "commit": "git-cz",
     "clean": "lerna exec 'rm -rf node_modules/ dist/ && npm prune' && rm -rf node_modules/ && npm prune",
-    "lint": "lerna run lint",
+    "lint": "lerna run lint --verbose",
     "lerna": "lerna",
     "prettier": "prettier './**/*.{js,jsx,ts,tsx,md,mdx}' --write",
     "prettier:check": "prettier --check '**/*.{jsx,js,ts,tsx,md,mdx}'",
     "prebuild": "lerna run prebuild",
     "prepublishOnly": "npm i && npm run build && npm run lint && npm run test",
-    "release": "lerna version --no-private --conventional-commits --create-release github --yes && lerna publish from-git --yes",
+    "lerna:version": "lerna version --no-private --conventional-commits --create-release github --yes",
+    "lerna:publish": "lerna publish from-git --yes",
     "start": "lerna run start",
     "test": "jest",
```

Comment:
maybe you want to call `lerna run test instead` this will also enable caching

Reply:
Yep. Working on that